.babelrc.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
```

Comment:
This file is the same as `.babelrc.cjs` except for the changes to the `@babel/preset-env` plugin configuration, and reformatting with Prettier.